react/tic-tac-toe/src/components/Game/components/Board/styles.module.scss

```diff
@@ -0,0 +1,9 @@
+.boardRow:after {
```

Comment:
I forgot to tell you about this! But usually we use double colon to differentiate between pseudo-elements and pseudo-selectors.